lib/src/app.dart

```diff
@@ -61,31 +61,13 @@ class _AppState extends State<App> with TrayListener {
           // background. This is a setting relevent to mobile devices.
           restorationScopeId: 'app',
 
-          // Provide the generated AppLocalizations to the MaterialApp. This
-          // allows descendant Widgets to display the correct translations
-          // depending on the user's locale.
-          localizationsDelegates: const [
-            AppLocalizations.delegate,
-            GlobalMaterialLocalizations.delegate,
-            GlobalWidgetsLocalizations.delegate,
-            GlobalCupertinoLocalizations.delegate,
-          ],
-          supportedLocales: const [
-            Locale('en', ''), // English, no country code
-            Locale('de', ''), // German, no country code
-            Locale('it', ''), // Italian, no country code
-            Locale('ru', ''), // Russian, no country code
-            Locale('pt', ''), // Portuguese, no country code
-            Locale('pt', 'BR'), // Brazilian Portuguese
-          ],
+          /// Setup for i18n, provider inherits from the device locale found
```

Comment:
nit: feels like `translations` instead of `i18n` is a clearer way to write these statements